docs/source/onnxruntime/usage_guides/models.mdx

```diff
@@ -30,16 +30,15 @@ pred = onnx_qa(question, context)
 ### Loading a vanilla Transformers model
 
 Because the model you want to work with might not be already converted to ONNX,  [`~optimum.onnxruntime.ORTModel`]
-includes a method to convert vanilla Transformers models to ONNX ones. Simply pass `from_transformers=True` to the
+includes a method to convert vanilla Transformers models to ONNX ones. Simply pass `export=True` to the
 [`~optimum.onnxruntime.ORTModel.from_pretrained`] method, and your model will be loaded and converted to ONNX on-the-fly:
 
 ```python
->>> from optimum.onnxruntime import ORTModelForSequenceClassification
+from optimum.onnxruntime import ORTModelForSequenceClassification
 
->>> # Load the model from the hub and export it to the ONNX format
->>> model = ORTModelForSequenceClassification.from_pretrained(
-...     "distilbert-base-uncased-finetuned-sst-2-english", from_transformers=True
-... )
+# Load the model from the hub and export it to the ONNX format
+model_id = "distilbert-base-uncased-finetuned-sst-2-english"
+model = ORTModelForSequenceClassification.from_pretrained(model_id, export=True)
```

Comment:
Why remove the `>>>` and `...`? They are the necessary for https://docs.python.org/3/library/doctest.html test to pass (which probably maybe does not...)

See: https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized

Reply:
This was removed for consistency with https://github.com/huggingface/optimum/blob/v1.7.3/docs/source/onnxruntime/usage_guides/models.mdx?plain=1#L112-L118, and because I think `>>>` makes the documentation harder to read, will close this PR as it was fixed in #935